lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -1305,8 +1305,10 @@ void HdVP2Mesh::_UpdateDrawItem(
         }
     }
 
-    if (itemDirtyBits & HdChangeTracker::DirtyVisibility) {
-        drawItemData._enabled = drawItem->GetVisible();
+    if (itemDirtyBits & HdChangeTracker::DirtyVisibility
+        || itemDirtyBits & HdChangeTracker::DirtyRenderTag) {
```

Comment:
Instead of using logical OR, the expression can be changed to use bitwise OR:

`if (itemDirtyBits  & (HdChangeTracker::DirtyVisibility | HdChangeTracker::DirtyRenderTag))`